Source/WebCore/platform/graphics/MediaPlayerPrivate.h

```diff
@@ -148,6 +148,7 @@ class MediaPlayerPrivateInterface {
     virtual void setPitchCorrectionAlgorithm(MediaPlayer::PitchCorrectionAlgorithm) { }
 
     virtual bool paused() const = 0;
+    virtual bool timeIsProgressing() const { return paused(); }
```

Comment:
That is strange, should it be `!paused()`?

Reply:
it should be yes !